index.yaml

```diff
@@ -33,6 +33,13 @@ indexes:
   - name: commit_message
   - name: commit_type
 
+- kind: DeleteExpCopiesRealtimeModel
```

Comment:
We don't seem to even have DeleteExpCopiesRealtimeModel in the codebase any more, right? If that's the case I think you can drop all entries related to it.

Reply:
Can you check whether, if you remove it and do stuff on the server, it gets regenerated?

I am wondering what autogenerated it. Does this model type actually exist (did I miss something)?